app/uk/gov/hmrc/brm/utils/CommonConstant.scala

```diff
@@ -34,6 +36,12 @@ object CommonConstant {
   val CONTENT_TYPE = "Content-Type"
   val CONTENT_TYPE_JSON = "application/json; charset=utf-8"
 
-
+  val INVALID_BIRTH_REFERENCE_NUMBER = Json.parse(
+    s"""
+       |{
+       |"code": "INVALID_BIRTH_REFERENCE_NUMBER",
+       |"reason": "The birth reference number does not meet the required length"
```

Comment:
@anujakhare Can we add this to the acceptance tests please for the BadRequest invalid BRN.